olive/packaging/packaging_generator.py

```diff
@@ -73,3 +75,37 @@ def _package_candidate_models(tempdir, footprint: Footprint, pf_footprint: Footp
         metric_path = str(model_dir / "metrics.json")
         with open(metric_path, "w") as f:
             json.dump(node.metrics.value, f)
+
+
+def _download_onnxruntime_package(cur_path):
+    try:
+        import onnxruntime
+    except ImportError:
+        logger.warning("onnxruntime is not installed, skip packaging onnxruntime package")
+        return
+
+    ort_version = onnxruntime.__version__
+
+    try:
+        # Download Python onnxruntime package
+        python_downlaod_path = str(cur_path / "sample_code" / "ONNXModel" / "python" / "ONNXRuntime")
+        downlaod_command = f"python -m pip download onnxruntime=={ort_version} --no-deps -d {python_downlaod_path}"
+        run_subprocess(downlaod_command)
+
+        # Download CPP onnxruntime package
+        cpp_ort_download_path = cur_path / "sample_code" / "ONNXModel" / "cpp" / "ONNXRuntime"
+        cpp_ort_download_path.mkdir(parents=True, exist_ok=True)
+        cpp_downlaod_path = str(cpp_ort_download_path / f"microsoft.ml.onnxruntime.{ort_version}.nupkg")
+        urllib.request.urlretrieve(
```

Comment:
https://onnxruntime.ai/docs/install/#install-onnx-runtime-ort-2 
Will this workable for c/c++/c# installation?

Reply:
No requested changes. Just raise it up for double confirm.

Reply:
Yeah the downloaded nuget package can be installed by dotnet command.

Reply:
Then, if it is better to use dotnet command? We may need to avoid use hard code like url in code as possible.

Reply:
`dotnet` command may not be able to download packages to local like `pip`. Url download is a way to download the package to local.